src/SFA.DAS.AssessorService.Application/Handlers/Apply/CreateApplicationHandler.cs

```diff
@@ -68,7 +68,7 @@ public async Task<Guid> Handle(CreateApplicationRequest request, CancellationTok
 
             return Guid.Empty;
         }
-
+         
```

Comment:
Remove this - blank lines are ok but not spaces

Reply:
removed